recipes/rdflib/meta.yaml

```diff
@@ -0,0 +1,75 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "rdflib" %}
+{% set version = "4.2.1" %}
+{% set sha256 = "eb02bd235606ef3b26e213da3e576557a6392ce103efd8c6c8ff1e08321608c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py<34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - isodate
+    - pyparsing
+
+  run:
+    - python
+    - isodate
+    - pyparsing
+
+test:
+  # Python imports
+  imports:
+    - rdflib
+    - rdflib/extras
```

Comment:
Having a `/` in the imports is not valid. Please check this. It looks like `conda skeleton` writes these path-like imports as well, but this is currently causing a build failure.